otter/convergence/composition.py

```diff
@@ -67,7 +67,7 @@ def json_to_LBConfigs(lbs_json):
         if lb.get('type') != 'RackConnectV3':
             lbd[lb['loadBalancerId']].append(CLBDescription(
                 lb_id=str(lb['loadBalancerId']), port=lb['port']))
-    return lbd
+    return dict(lbd)
```

Comment:
Looks like you're just trying to get rid of the `defaultdict` behavior, right? I guess maybe one day this could be a pmap...